package.json

```diff
@@ -92,9 +92,9 @@
     }
   },
   "dependencies": {
-    "@sentry/node": "^7.12.1",
-    "@sentry/tracing": "^7.12.1",
-    "@sentry/utils": "^7.12.1",
+    "@sentry/node": "^7.22.0",
```

Comment:
🤔 maybe it's safer to fix an exact version here since the issue appeared in a minor version update.